e2e/app/page/cohort-participant-detail-page.ts

```diff
@@ -20,8 +21,46 @@ export default class CohortParticipantDetailPage extends AuthenticatedPage {
     return true;
   }
 
+  // get the particpant ID displayed between the two pi-angle button on participant detail page
+  async getParticipantIDnum(): Promise<string> {
+    const selector = `//*[@class="detail-header"]//span[contains(., "Participant ")]`;
+    const pID = await this.extractParticipantID(selector);
+    console.log(`Participant ID: ${pID}`);
+    return pID;
+  }
+
+  // extract only the number of the participant
+  async extractParticipantID(selector: string): Promise<string> {
+    const elemt = await this.page.waitForXPath(selector, {visible: true});
+    const textContent = await getPropValue<string>(elemt, 'textContent');
+    const regex = new RegExp(/\d+/);
+    return (regex.exec(textContent))[0];
+  }
+  
+  // get the back to review set button
   async getBackToReviewSetButton(): Promise<Button> {
     return Button.findByName(this.page, {name: LinkText.BackToReviewSet});
   }
+ 
+   // click on the pen icon located on the side bar
+   async clickPenIconHelpSideBar(): Promise<void> {
+    const clickPenIcon = await this.page.waitForXPath(`//*[@data-test-id="help-sidebar-icon-annotations"]`);
```

Comment:
are you waiting for a visible element? if yes, need `{visible: true}`.

Reply:
![image](https://user-images.githubusercontent.com/48066285/99705061-db54eb00-2a5e-11eb-9796-4660a542efcb.png)
yes, this icon that's on sidebar on participant detail page

Reply:
Done!